packages/edit-site/src/components/global-styles/root-menu.js

```diff
@@ -33,10 +35,20 @@ function RootMenu() {
 	const hasShadowPanel = true; // useHasShadowPanel( settings );
 	const hasDimensionsPanel = useHasDimensionsPanel( settings );
 	const hasLayoutPanel = hasDimensionsPanel;
+	const hasBackgroundPanel = useHasBackgroundPanel( settings );
```

Comment:
The background panel doesn't show at all for me at the moment. I think we should be using `rawSettings` here, not the return value of `useSettingsForBlockElement()`

```suggestion
	const hasBackgroundPanel = useHasBackgroundPanel( rawSettings );
```